spec/support/corpus.rb

```diff
@@ -29,9 +29,11 @@ class Project
       START_MESSAGE               = 'Starting - %s'.freeze
       FINISH_MESSAGE              = 'Mutations - %4i - %s'.freeze
 
+      DEFAULT_MUTATION_COUNT = 0
```

Comment:
Some function somebody passes an anonymous value in to add some debuging or tracing mecnahism to *all* objects passing by.

You could even violate basic math:

> What would that even look like?

Actually recent ruby forbids opening the singleton class for `Fixums` nice.